VideoLocker/src/main/java/org/edx/mobile/view/adapters/LectureAdapter.java

```diff
@@ -106,17 +106,17 @@ public BaseViewHolder getTag(View convertView) {
                 .findViewById(R.id.bulk_download_layout);
         holder.download_pw = (ProgressWheel) convertView.
                 findViewById(R.id.progress_wheel);
-        holder.progresslayout = (LinearLayout) convertView
+        holder.progress_layout = (LinearLayout) convertView
                 .findViewById(R.id.download_progress);
         return holder;
     }
 
     private static class ViewHolder extends BaseViewHolder {
-        TextView lectureTitle;
+        TextView lecture_title;
```

Comment:
I'm glad to see this being made consistent, but we should be using camelCase instead of snake_case.